C7Engine/TileExtensions.cs

```diff
@@ -0,0 +1,18 @@
+namespace C7Engine
+{
+	using C7GameData;
+
+	public static class TileExtensions {
```

Comment:
I'd like to hear your thoughts on extension methods in #238, since my only experience with them is as a way to extend third party classes. What do they provide being in the same namespace as the data classes?